fast/t13.hoc

```diff
@@ -53,5 +53,5 @@ strdef s
 //b = b && (strcmp(s, "") == 0)
 {load_file("compare.hoc")}
 if (strcmp(nrnversion(8), "arm64-Darwin") != 0) { // use_long_double(1) same as double
-  b = b && (compare_files("temp13", "t13.stdout", 1e-8) || compare_files("temp13", "t13.stdout.c3", 1e-8) || compare_files("temp13", "t13.stdout.ci", 1e-8) || compare_files("temp13", "t13.stdout.ci.nvhpc", 1e-8))
+  b = b && (compare_files("temp13", "t13.stdout", 1e-8) || compare_files("temp13", "t13.stdout.c3", 1e-8) || compare_files("temp13", "t13.stdout.ci", 1e-8) || compare_files("temp13", "t13.stdout.ci.nvhpc", 1e-8) || compare_files("temp13", "t13.stdout.ci.nvhpc.21.9", 1e-8))
```

Comment:
just curious, how different are these ref results compared to recent `t13.stdout.ci.nvhpc` ?